src/components/Clock/Clock.tsx

```diff
@@ -9,7 +11,7 @@ type State = {
 };
 
 export class Clock extends React.Component<Props> {
-  timerId: NodeJS.Timer | null = null;
+  timerId?: NodeJS.Timer = undefined;
```

Comment:
why do you need here  = undefined?

Reply:
yeah, thaths my bad? that value sets by default